chapters/002-smol-internet/README.md

```diff
@@ -190,251 +149,205 @@ Let's go back to our `docker-compose.yml` and give `boudi` an additional network
-19:57:13.285392 IP 10.1.2.2 > 10.1.2.3: ICMP echo reply, id 4, seq 1, length 64
-19:57:14.321680 IP 10.1.2.3 > 10.1.2.2: ICMP echo request, id 4, seq 2, length 64
-19:57:14.321698 IP 10.1.2.2 > 10.1.2.3: ICMP echo reply, id 4, seq 2, length 64
-19:57:15.342833 IP 10.1.2.3 > 10.1.2.2: ICMP echo request, id 4, seq 3, length 64
-19:57:15.342849 IP 10.1.2.2 > 10.1.2.3: ICMP echo reply, id 4, seq 3, length 64
-19:57:16.367010 IP 10.1.2.3 > 10.1.2.2: ICMP echo request, id 4, seq 4, length 64
-19:57:16.367026 IP 10.1.2.2 > 10.1.2.3: ICMP echo reply, id 4, seq 4, length 64
-^C
-8 packets captured
-8 packets received by filter
-0 packets dropped by kernel
+18:53:54.166970 ARP, Request who-has 10.1.2.3 tell 10.1.2.2, length 28
+18:53:54.167080 ARP, Request who-has 10.1.2.2 tell 10.1.2.3, length 28
+18:53:54.167123 ARP, Reply 10.1.2.2 is-at 02:42:0a:01:02:02, length 28
+18:53:54.167168 ARP, Reply 10.1.2.3 is-at 02:42:0a:01:02:03, length 28
 ```
 
-We can see that the ping from `boudi` made it to `tara`! Huzzah!
-
-### Can `tara` ping `boudi`?
-
-Now... let's see if `tara` is able to ping `boudi`. We're going to do the same 3 window setup, but this time `tara` will be running both `ping` and `tcpdump`.
-
-The interesting thing with `tara` pinging `boudi` is that `boudi` has 2 network IP addresses we can ping. Which one should we use? Well, 10.1.1.3 is the IP that's on the `squasheeba` network. `tara`, unfortunately, doesn't know anything about how to reach this network, which means, if we try to ping it, `boudi` won't receive the ping and `tara` is just screaming into the void. If instead, we use `boudi`'s IP on `doggonet`, 10.1.2.3, we should have a successful result.
+We go over this in more detail in [ip-and-mac-addresses.md in the appendix](../appendix/ip-and-mac-addresses.md), but let's look at a high level at what's going on here. IP addresses, like `10.1.2.3`, are used by machines for identifying where packets should be routed across an internet. So what we've been working with so far is designed to help machines communicate when there are multiple networks. HOWEVER. Within a network, machines are not identified by an IP address, but instead by a MAC address. In order for packets to be forwarded from one machine on a network to another machine on the same network, each machine needs to discover the MAC address that corresponds to the IP address identified in the packets. ARP, or Address Resolution Protocol, is the process by which this is done.
 
-1. `hopon tara` will run `ping 10.1.2.3`
-2. `hopon tara` will run `tcpdump -ne`
-3. `hopon boudi` will run `tcpdump -ni eth0`
-
-> **NOTICE**
-> We added the `-e` flag to our `tcpdump` command for `tara`. Why? That flag reveals information about ethernet headers in each packet. If we look at the network interface information for `boudi`, we can see the mac address for its connection on `doggonet`: 02:42:0a:01:02:03.
+Let's read what's happening with the `ARP` requests we see above:
```

Comment:
Should we have a central place where we talk about the commands we use and the flags we use with them?

Reply:
I like this idea. I think we should leave the more detailed explanations in the chapter, but then we can have a reference guide for people to use for findability...

I was also thinking that at some point I wanted to include links to [specific comix from Julia Evans](https://wizardzines.com/comics/) in our explanations. For example:
* [ping & traceroute](https://wizardzines.com/comics/ping/)
* [tcpdump](https://wizardzines.com/comics/tcpdump/)
* maybe something on [containers](https://wizardzines.com/comics/why-containers/)
* we reference [iptables](https://wizardzines.com/comics/iptables/) in chapter 003

Maybe we can add a `## Helpful Links` section to our chapters? Or actually do the work to figure out where the best place to link these things is 😆

Reply:
@tundal45  and I talked about this and i'll make another PR that includes a reference guide with a first stab and linking to it where useful.